corehq/apps/domain/views/accounting.py

```diff
@@ -1947,3 +1950,74 @@ def pause_subscription(request, domain):
     return HttpResponseRedirect(
         reverse(DomainSubscriptionView.urlname, args=[domain])
     )
+
+
+@require_POST
+@login_and_domain_required
+@require_permission(HqPermissions.edit_billing)
+def enable_subscription_auto_renew(request, domain):
+    current_subscription = Subscription.get_active_subscription_by_domain(domain)
```

Comment:
I'm worried that this may be too indirect. Presumably when I issue the call to enable auto-renew, I mean to do it on a specific subscription, right? While unlikely, it seems possible that the `current_subscription` fetched when this call is executed does not match the subscription the user issued this call for. Would it be possible to pass in the subscription ID to ensure that the two line up?